website/docs/r/netapp_backup_policy.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: netapp_backup_policy"
+description: |-
+  Manages a NetApp Backup Policy.
+---
+
+# netapp_backup_policy
+
+Manages a NetApp Backup Policy.
+
+## NetApp Backup Policy Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "example-netappaccount"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_netapp_backup_policy" "example" {
+  name                    = "example-netappbackuppolicy"
+  resource_group_name     = azurerm_resource_group.example.name
+  location                = azurerm_resource_group.example.location
+  account_name            = azurerm_netapp_account.example.name
+  daily_backups_to_keep   = 2
+  weekly_backups_to_keep  = 2
+  monthly_backups_to_keep = 2
+  enabled                 = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Backup Policy. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group where the NetApp Backup Policy should be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `account_name` - (Required) The name of the NetApp account in which the NetApp Policy should be created under. Changing this forces a new resource to be created.
+
+* `daily_backups_to_keep` - (Optional) Provides the number of daily backups to keep, defaults to `2` which is the minimum, maximum is 1019.
+
+* `weekly_backups_to_keep` - (Optional) Provides the number of weekly backups to keep, defaults to `1` which is the minimum, maximum is 1019.
+
+* `monthly_backups_to_keep` - (Optional) Provides the number of monthly backups to keep, defaults to `1` which is the minimum, maximum is 1019.
+
+* `enabled` - (Optional) Whether the Backup Policy is enabled. Defaults to `true`.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource.
+
+~> **Note:** Currently, the combined (daily + weekly + monthy) retention counts cannot exceed 1019.
```

Comment:
This note should be moved to just below `monthly_backups_to_keep` so that way the note is grouped with what it is talking about